tests/test_orient_sync_voting.py

```diff
@@ -161,3 +195,13 @@ def test_command_line():
         )
         # check that the command completed successfully
         assert result.exit_code == 0
+
+
+def check_rotations(rots_est, rots_gt):
```

Comment:
I just made it to reduce code duplication within this test, but it would probably be nice as a public function anywhere we do diagnostics on estimated rotations (ie. other tests, tutorials,...)

Reply:
Ok, left this as public and moved it to `utils/coor_trans.py` where the other global alignment methods are. Implemented it in all relevant places and added a test.

Reply:
Awesome,thanks!